examples/creatives/banner_nonlinear.js

```diff
@@ -20,8 +20,8 @@ class BannerNonLinear extends BaseSimidCreative {
      */
     addButtonClickActions_() {
         this.sendMessageOnButtonClick_("close_ad", CreativeMessage.REQUEST_STOP);
-        this.sendMessageOnButtonClick_("ad_text", CreativeMessage.REQUEST_EXPAND);
-        this.sendMessageOnButtonClick_("minimize_ad", CreativeMessage.REQUEST_COLLAPSE);
+        this.onExpand_();
```

Comment:
if there's a function already meant to send message on-click, why not just utilize that and put your css portions of the code in a callback function instead of creating these 2 new functions?

Reply:
I'm not sure what you mean by that- how would that look?

Reply:
using the sendMessageOnButtonClick_ function and then just perform your classAdds in => function

Reply:
that's a great idea! Thanks